test/test_hypothesis.py

```diff
@@ -100,10 +101,11 @@ def test_register_default_strategies():
     from hypothesis_jsonschema._impl import STRING_FORMATS
 
     assert "binary" in STRING_FORMATS
+    assert "byte"
```

Comment:
this one should be `assert "byte" in STRING_FORMATS`

Reply:
:man_facepalming: